tests/en/homeassistant_HassDecreaseTimer.yaml

```
@@ -0,0 +1,37 @@
+---
+language: en
+tests:
+  - sentences:
+      - "remove 5 minutes from timer"
+    intent:
+      name: HassDecreaseTimer
+      slots:
+        minutes: 5
+    response: Removed time
+
+  - sentences:
+      - "remove 5 minutes from 1 hour timer"
+    intent:
+      name: HassDecreaseTimer
+      slots:
+        minutes: 5
+        start_hours: 1
+    response: Removed time
+
+  - sentences:
+      - "remove 5 minutes from pizza timer"
+    intent:
+      name: HassDecreaseTimer
+      slots:
+        minutes: 5
+        name: "pizza "
```

Shouldn't this not match the trailing space? 
```suggestion
        name: "pizza"
```

---

The wildcard consumes the trailing space in hassil. This will be fixed in a larger PR later that will touch on multiple languages.